src/routes/safe/components/Transactions/TxsTable/columns.tsx

```diff
@@ -30,15 +29,15 @@ const NOT_AVAILABLE = 'n/a'
 interface AmountData {
   decimals?: number | string
   symbol?: string
-  value: number | string
+  value: string
 }
 
 const getAmountWithSymbol = (
   { decimals = 0, symbol = NOT_AVAILABLE, value }: AmountData,
   formatted = false,
 ): string => {
-  const nonFormattedValue = new BigNumber(value).times(`1e-${decimals}`).toFixed()
-  const finalValue = formatted ? formatAmount(nonFormattedValue).toString() : nonFormattedValue
+  const humanReadableAmount = humanReadableTokenAmount(value, decimals as number)
+  const finalValue = formatted ? formatAmount(humanReadableAmount).toString() : humanReadableAmount
```

Comment:
It was already there, but why do we need to convert `toString` the value returned by `formatAmount`?

Reply:
yeah, just copied and pasted but didn't make sense anymore, I removed it